src/RetailRedOmniEnablement/Resources/config.xml

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Lets name the folder `RetailredStorefront` inside src folder. Same as done for magento plugins